openmetadata-spec/src/main/resources/json/schema/entity/services/connections/dashboard/powerBIConnection.json

```diff
@@ -59,6 +59,12 @@
       },
       "default": ["https://analysis.windows.net/powerbi/api/.default"]
     },
+    "pagination_entity_per_page": {
+      "title": "Pagination Entity Per Page",
+      "description": "Entity Limit set here will be used to paginate the PowerBi APIs",
+      "type": "integer",
+      "default": 100
+    },
```

Comment:
as we are paginating the apis of power bi itself, imo it will loop it that many times with the next token it fetches, so instead of asking the user to provide the hardcoded pagination number and changing the json schema, having a hardcoded pagination number should suffice..
wdyt?